src/WingetCreateTests/WingetCreateTests/UnitTests/PackageParserTests.cs

```diff
@@ -48,7 +49,7 @@ public void ParseExeInstallerFile()
 
             Manifests manifests = new Manifests();
 
-            Assert.IsTrue(PackageParser.ParsePackages(new[] { testExeInstallerPath }, new[] { TestConstants.TestExeInstaller }, manifests));
+            Assert.IsTrue(PackageParser.ParsePackages(new[] { testExeInstallerPath }, new[] { TestConstants.TestExeInstaller }, manifests, out List<PackageParser.DetectedArch> detectedArchOfInstallers));
```

Comment:
Since these out vars are never used, use a discard instead:

, out _)

Reply:
Ah I didn't know I could do that, much cleaner now, changed to suggested.